lib/connection/result/result.js

```diff
@@ -694,20 +634,29 @@ Result.prototype.getColumn = function (columnIdentifier)
  * Returns the statement id generated by the server for the statement that
  * produced this result.
  *
+ * @deprecated
```

Comment:
let's add a comment above that `getQueryId` should be used instead

Reply:
Applied.